scripts/test-SSLv3-padding.py

```diff
@@ -132,6 +134,70 @@ def main():
     conversations["zero-filled padding in SSLv3"] = \
             conversation
 
+    # max size SSLv3 padding
+    conversation = Connect(host, port)
+    node = conversation
+    ciphers = [CipherSuite.TLS_RSA_WITH_AES_128_CBC_SHA,
+               CipherSuite.TLS_EMPTY_RENEGOTIATION_INFO_SCSV]
+    node = node.add_child(ClientHelloGenerator(ciphers, version=(3, 0)))
+    node = node.add_child(ExpectServerHello())
+    node = node.add_child(ExpectCertificate())
+    node = node.add_child(ExpectServerHelloDone())
+    node = node.add_child(ClientKeyExchangeGenerator())
+    node = node.add_child(ChangeCipherSpecGenerator())
+    node = node.add_child(FinishedGenerator())
+    node = node.add_child(ExpectChangeCipherSpec())
+    node = node.add_child(ExpectFinished())
+    text = b"GET / HTTP/1.0\nX-bad: aaaa\n\n"
+    hmac_tag_length = 20
+    block_size = 16
+    # make sure that padding has full block to work with
+    assert (len(text) + hmac_tag_length) % block_size == 0
+    node = node.add_child(fuzz_padding(ApplicationDataGenerator(text),
+                                       min_length=128//8))
```

Comment:
to verify my understanding of the test, the min_length=16, means that the maximum pad value will be 15.

Reply:
if you consider the length byte of padding to be separate from "padding" then yes, it will have 15 bytes of "padding"